sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -1468,8 +1468,8 @@
     <module>../../core/azure-core-test</module>
     <module>../../core/azure-core-tracing-opentelemetry</module>
     <module>../../cosmos/azure-cosmos</module>
-    <module>../../cosmos/azure-cosmos-benchmark</module> <!-- CosmosSkip - Benchmark will go away, this is temporary -->
-    <module>../../cosmos/azure-cosmos-examples</module> <!-- CosmosSkip - Examples will go away, this is temporary -->
+    <module>../../cosmos/azure-cosmos-benchmark</module>
+    <module>../../cosmos/azure-cosmos-examples</module> 
```

Comment:
@mitchdenny please do not remove these comments from the pom file. These are the responsibility of the Cosmos team to remove before GA and I want them kept here.

Reply:
Put them back in, although for the most part the module list in the parent POMs can be completely deleted (unless we have a script somewhere that is still depending on them). They aren't considered at all for the build itself anymore. Only the ```sdk/cosmos/pom.xml``` defines the modules that are used in the Cosmos build.